docs/topics/feed-exports.rst

```diff
@@ -185,6 +185,10 @@ passed through the following settings:
  * :setting:`AWS_ACCESS_KEY_ID`
  * :setting:`AWS_SECRET_ACCESS_KEY`
 
+You can also define a custom ACL for exported feeds using this setting:
+
+ * :setting:`FEED_STORAGE_S3_ACL`
```

Comment:
I think we should consider changing the name to `FEED_ACL`, and follow an approach similar to other feed storage settings here such as `FEED_URI`: The backend-specific documentation is provided on the backend-specific documentation.

Reply:
Oh, I see this approach is already followed in other places. I guess any change to that is then a different beast to treat in a different changeset. Lets merge.